app/javascript/packages/build-sass/cli.js

```diff
@@ -20,9 +21,10 @@ const flags = args.filter((arg) => arg.startsWith('-'));
 
 const isWatching = flags.includes('--watch');
 const outDir = flags.find((flag) => flag.startsWith('--out-dir='))?.slice(10);
-const loadPaths = flags
-  .filter((flag) => flag.startsWith('--load-path='))
-  .map((flag) => flag.slice(12));
+const loadPaths = [
+  ...getDefaultLoadPaths(),
+  ...flags.filter((flag) => flag.startsWith('--load-path=')).map((flag) => flag.slice(12)),
```

Comment:
WDYT of making `'--load-path='` a constant or something, so that we could remove the 12 as a magic number?
```suggestion
  ...flags.filter((flag) => flag.startsWith(LOAD_PATH_FLAG)).map((flag) => flag.slice(LOAD_PATH_FLAG.length)),
```

Reply:
I did a thing in a19d8f8 but I'm not convinced I didn't just make a bigger mess 😅

Reply:
ಠ_ಠ well... I would at least maybe move that into a separate function instead of just doing it inline, but otherwise ya it seems reasonable since there are 3 args now

```
function parseArguments(argv, ['watch', 'out-dir', 'load-path']): { keyof(args): string } {
  ...
}
```

Reply:
Rather that rolling a custom `parseArgs`, I think if eventually we'd want to move to the native `parseArgs`, what I might do is pull in a [polyfill](https://github.com/pkgjs/parseargs), and we can drop that once we're on Node 18.

Reply:
And actually, now that I think of it, since this is a published package, we'd want to use a polyfill until we're comfortable making Node v18 the minimum supported version for all consumers, which might be something we do a little later than running v18 oursevles.

Reply:
Ok, pushed up `parseArgs` in 874df80a886a33e92ad1625bdaef02d40669aa9c. It worked pretty well IMO!